### DISCUSSION TITLE
can I build on pure OpenWRT 19.07.xx for my router, cpu MediaTek MT7620A

### DISCUSSION THREAD

**Discussion by centrsvyaz** [Ysurac/openmptcprouter#1915] [General]
where do I begin?

**Comment by Ysurac** [comment]
19.07 is not supported anymore: too old kernel.

**Comment by centrsvyaz** [comment]
Linux  4.14.215 #0 Tue Jan 19 13:10:02 2021 mips GNU/Linux

i can upgrade kernel to 5.4 on 07.19.xx  ?

**Reply by centrsvyaz** [reply]
OpenWrt 21.02  5.4  do i need to start with this version?

**Reply by Ysurac** [reply]
yes

**Reply by centrsvyaz** [reply]
how much space do you need on flash memory?
64, 128, 512 MB?

**Reply by Ysurac** [reply]
If you don't use OpenMPTCProuter, I have no idea.

**Reply by centrsvyaz** [reply]
I want to install OpenMPTCProuter on an unsupported router

**Reply by Ysurac** [reply]
Did you read https://github.com/Ysurac/openmptcprouter/wiki/Create-image-for-unsupported-platform ? This need less than 200MB

**Reply by centrsvyaz** [reply]
can I build for MediaTek MT7621A, according to this principle from your link?

**Reply by Ysurac** [reply]
Yes. Ubiquiti ER-X (that is supported by OpenMPTCProuter, even if still alpha and not really stable for some users) is also based on MT7621.

**Reply by centrsvyaz** [reply]
Ubiquiti ER-X MT7621АТ
